### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean [hunk 1]
```diff
@@ -736,24 +733,27 @@ theorem cos_pi_over_two_pow : ∀ n : ℕ, cos (π / 2 ^ (n + 1)) = sqrtTwoAddSe
...
+    rw [eq_div_iff_mul_eq this, eq_comm, sqrtTwoAddSeries, sqrt_eq_iff_sq_eq, mul_pow, cos_sq,
+      ← mul_div_assoc, pow_succ, mul_div_mul_left _ _ this, cos_pi_over_two_pow _, add_mul]
+    · congr
+      · norm_num
+      · rw [mul_comm, sq, mul_assoc, ← mul_div_assoc, mul_div_cancel_left _ this, ← mul_div_assoc,
+          mul_div_cancel_left _ this]
+    · exact add_nonneg two_pos.le (sqrtTwoAddSeries_zero_nonneg _)
+    apply le_of_lt
+    apply mul_pos _ two_pos
+    apply cos_pos_of_mem_Ioo ⟨_, _⟩
```
Comment (ADedecker) [inline Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean:745]: Can't you merge these into a single `refine`?

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean [hunk 2]
```diff
@@ -736,24 +733,27 @@ theorem cos_pi_over_two_pow : ∀ n : ℕ, cos (π / 2 ^ (n + 1)) = sqrtTwoAddSe
...
+        apply pi_div_two_pos
+      · apply div_pos pi_pos
+        apply pow_pos
+        norm_num
+    apply div_lt_div' (le_refl π) _ pi_pos two_pos
     refine' lt_of_le_of_lt (le_of_eq (pow_one _).symm) _
-    apply pow_lt_pow; norm_num; apply Nat.succ_lt_succ; apply Nat.succ_pos; all_goals norm_num
+    apply pow_lt_pow
+    · norm_num
+    · apply Nat.succ_lt_succ; apply Nat.succ_pos
```
Comment (ADedecker) [inline Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean:756]: Same here

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean [hunk 3]
```diff
@@ -764,90 +764,97 @@ theorem sin_sq_pi_over_two_pow (n : ℕ) :
...
+  rw [eq_div_iff_mul_eq two_ne_zero, eq_comm, sqrt_eq_iff_sq_eq, mul_pow,
+    sin_sq_pi_over_two_pow_succ, sub_mul]
   · congr <;> norm_num
   · rw [sub_nonneg]
     apply le_of_lt
     apply sqrtTwoAddSeries_lt_two
-  apply le_of_lt; apply mul_pos; apply sin_pos_of_pos_of_lt_pi
+  apply le_of_lt
+  apply mul_pos ?_ two_pos
+  apply sin_pos_of_pos_of_lt_pi
```
Comment (ADedecker) [inline Mathlib/Analysis/SpecialFunctions/Trigonometric/Basic.lean:784]: Same here

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by ADedecker** [comment]
bors retry

**Pull request closed without merging**